core/scripts/others/helper.py

```diff
@@ -0,0 +1,107 @@
+"""@package module
+Helper functions to be used through the tool
+"""
+
+import os
+import shutil
+
+
+def linux_path(path):
+    """Converts Windows style path to linux style path"""
+
+    path = path.replace("\\", "/")
+
+    return path
+
+
+def fill_template(string, data):
+    """Fills the template based on the data provided"""
+
+    string = string.replace("\\n", "\n").replace("\\t", "\t")
+    for key in data:
+        if type(data[key]) == list:
+            value = "\"" + data[key][0] + "\""
+        else:
+            value = data[key]
+
+        string = string.replace("{{" + key + "}}", value)
```

Comment:
You could use a regex to match white spaces `{{key}}` and `{{ key }}` something like `"{{.*" + key + ".*}}"`

Reply:
will make that change